hivemind/p2p/p2p_daemon_bindings/control.py

```diff
@@ -107,8 +110,14 @@ async def create(
         daemon_connector: DaemonConnector,
         listen_maddr: Multiaddr = Multiaddr(DEFAULT_LISTEN_MADDR),
         use_persistent_conn: bool = True,
+        persistent_conn_max_msg_size=2 << 22,
```

Comment:
```suggestion
        persistent_conn_max_msg_size: int = DEFAULT_MAX_MSG_SIZE,
```

The typo annotation is missing.